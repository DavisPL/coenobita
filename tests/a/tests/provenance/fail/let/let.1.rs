#![allow(unused_variables)]
#![allow(unused_assignments)]

fn main() {
    #[cnbt::provenance((bin,b))]
    let w = 5;
}