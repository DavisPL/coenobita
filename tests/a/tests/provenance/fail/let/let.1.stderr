error: expected (bin,b), found (bin,bin)
 --> tests/provenance/fail/let/let.1.rs:6:13
  |
6 |     let w = 5;
  |             ^
